contentgrid-spring-data-rest/src/test/java/com/contentgrid/spring/data/rest/webmvc/HalFormsProfileControllerTest.java

```diff
@@ -0,0 +1,170 @@
+package com.contentgrid.spring.data.rest.webmvc;
+
+import com.contentgrid.spring.test.fixture.invoicing.InvoicingApplication;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.hateoas.MediaTypes;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+@SpringBootTest
+@ContextConfiguration(classes = {
+        InvoicingApplication.class,
+        ContentGridSpringDataRestProfileConfiguration.class
+})
+@AutoConfigureMockMvc(printOnlyOnFailure = false)
+class HalFormsProfileControllerTest {
+
+    @Autowired
+    MockMvc mockMvc;
+
+    @Test
+    void profileController_embeddedContent() throws Exception {
+        mockMvc.perform(MockMvcRequestBuilders.get("/profile/customers")
+                .accept(MediaTypes.HAL_FORMS_JSON)
+        ).andExpect(MockMvcResultMatchers.status().isOk())
+                .andExpect(MockMvcResultMatchers.content().json("""
+                        {
+                            _links: {
+                                describes: [
+                                    {
+                                        name: "collection",
+                                        href: "http://localhost/customers"
+                                    },
+                                    {
+                                        name: "item",
+                                        href: "http://localhost/customers/{id}",
+                                        templated: true
+                                    }
+                                ]
+                            },
+                            _templates: {
+                                "create-form": {
```

Comment:
I would just call it `"create"` ?

Reply:
I used `create-form`, because it is a registered link relation, which description matches what we use the hal-forms template for.
I know this is a hal-forms template name, and technically not a link relation, but I found it close enough to prefer to use that one.